docs/routes.md

```diff
@@ -0,0 +1,268 @@
+
+### Page Data
+
+GETs and PUTs to pages work similarly to components. API calls without an extension will update/return data for that page, while the `.json` and `.html` extensions will return the composed (page and layout, and their child components) JSON and HTML, respectively.
+
+### Publishing
+
+Publishing a page has a special convenience behavior where all components referenced by the page will also be published.  (Example: https://github.com/nymag/amphora/pull/78)
+
+## URIs
+
+```
+/uris/<base64(:path)>
+```
+
+A URI is used to redirect some slug or URI to another page or component.  They can also redirect to other uris (establishing a 301 redirect), or several uris can point to the same resource.  URIs are stored as Base64, so:
+
+- `example.com` is `/uris/ZXhhbXBsZS5jb20=` => `/pages/jdskla@published`
+- `example.com/other/` is `/uris/ZXhhbXBsZS5jb20vb3RoZXI=` => `/pages/4revd3s@published`
+
+A URI is assumed to be pointing at the `@published` version if another version is not provided.  Therefore, only published content or specially tagged versions can be publicly exposed through URIs.
+
+## Lists
+
+```
+/lists
+/lists/:id
+```
+
+Lists are a _temporary_ solution until search functionality is discussed.  It is currently used for the tags component and the autocomplete behavior as a temporary place to store lists of information.  It can store any list of data on a `PUT`, and return back the same list of data with a `GET`.
+
+## Users
+
+### Overview
+```
+/users
+/users/:id
+```
+
+Each user has a `username` and `provider`, which determines how they authenticate over oauth. Users can also have other data, including `name`, `imageUrl`, and `title`.
+
+## Schedule
+
+```
+/schedule
+/schedule/:id
+```
+
+Schedule is a list of pages or components that will be published in the future.  Items in this list will be published when the time passes.  Each item has two properties: `at`, which is a UNIX timestamp, and the `publish` property that is a reference to what will be published after that time.
+
+When an item is scheduled, a `@scheduled` version is created on the item that will be published.  When the item is eventually published, the `@scheduled` version will be removed.  The `@scheduled` version is a reference to the actual schedule URI that can be deleted to cancel the scheduled publication.
+
+### List of scheduled items
+
+`GET /schedule` will return a list of scheduled items, such as:
+```json
+[
+  {
+    "_ref": "domain.com/some-path/schedule/3f-abc",
+    "at": 44392893402093,
+    "publish": "abc"
+  }
+]
+```
+
+### Add a scheduled item
+
+`POST /schedule` will add an item to be published in the future. The format of the item should be of
+```json
+{
+  "at": 44392893402093,
+  "publish": "<uri to be published>"
+}
+```
+where the `at` is a UNIX timestamp and the `publish` is a ref to a page or component.  On success, it will return a 201 with a reference to the schedule URI that was created.
```

Comment:
```suggestion
where the `at` is a UNIX timestamp and the `publish` is a ref to a page or component. On success, it will return a 201 with a reference to the schedule URI that was created.
```